src/bin/ion/commands/stats.rs

```diff
@@ -22,14 +22,18 @@ impl IonCliCommand for StatsCommand {
     }
 
     fn about(&self) -> &'static str {
-        "Print the analysis report of the input data stream, including the total number of top-level values, their minimum, maximum, and mean sizes,\n\
-        and plot the size distribution of the input stream. The report should also include the number of symbol tables in the input stream,\n\
-        the total number of different symbols that occurred in the input stream, and the maximum depth of the input data stream.\n\
-        Currently, this subcommand only supports data analysis on binary Ion data."
+        "Print statistics about an Ion stream"
     }
 
     fn configure_args(&self, command: Command) -> Command {
-        command.with_input()
+        command
+            .long_about("Print the analysis report of the input data stream, including the total number of\n\
+        top-level values, their minimum, maximum, and mean sizes, and plot the size distribution of\n\
+        the input stream. The report should also include the number of symbol tables in the input\n\
+        stream, the total number of different symbols that occurred in the input stream, and the\n\
+        maximum depth of the input data stream. Currently, this subcommand only supports data\n\
+        analysis on binary Ion data.")
```

Comment:
looks like a good use for `indoc`: https://github.com/dtolnay/indoc

Reply:
I don't know. I actually think that something like https://crates.io/crates/textwrap would be potentially more useful so that we don't have to worry about line wrapping any of our help text, but I'm saving that for another day.